docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+---
+displayed_sidebar: "English"
+---
+
+# Query Profile Structure and Metrics
+
+## Structure of Query Profile
+
+The structure of a Query Profile is closely related to the design of StarRocks' execution engine and consists of the following five parts:
+
+- **Fragment**: Execution tree. A query is composed of one or more fragments.
+- **FragmentInstance**: Each fragment can have multiple instances, each instance is called a FragmentInstance, and is executed by different computing nodes.
+- **Pipeline**: A FragmentInstance is split into multiple pipelines. A pipeline is an execution chain consisting of a group of connected Operator instances.
+- **PipelineDriver**: A Pipeline can have multiple instances, each instance is called a PipelineDriver, to fully utilize multiple computing cores.
+- **Operator**: A PipelineDriver consists of multiple Operator instances.
+
+![profile-3](../assets/profile-3.png)
+
+### Query Profile Merging Strategy
+
+By analyzing the structure of Query Profile, you can easily observe that multiple FragmentInstances associated with the same Fragment have a high degree of similarity in structure. Similarly, multiple PipelineDrivers belonging to the same Pipeline also exhibit similar structural features. To reduce the volume of the Query Profile, StarRocks by default merges the FragmentInstance layer and the PipelineDriver layer. As a result, the original five-layer structure is simplified to three layers:
+
+- Fragment
+- Pipeline
+- Operator
+
+You can control this merging behavior through a session variable `pipeline_profile_level`, which has two possible values:
+
+- `1` (Default): StarRocks merges the metrics into a three-layer structure.
+- `2`: StarRocks does not merge the metrics. The original five-layer structure is retained.
+- Any other value will be treated as the default value `1`.
+
+Generally, we do not recommend setting this parameter to `2` because the Query Profile with the five-layer structure faces numerous limitations. For example, you cannot visualize it for analysis using any tools. Therefore, unless the merging process leads to the loss of crucial information, you do not need to adjust this parameter.
```

Comment:
1. faces numerous limitations. ---has many limitations.    2. you cannot perform visualized analysis on the profile using any tools